Content.Server/Anomaly/Effects/LiquidAnomalySystem.cs

```diff
@@ -0,0 +1,128 @@
+using System.Linq;
+using Content.Server.Anomaly.Components;
+using Content.Server.Chemistry.Components.SolutionManager;
+using Content.Server.Chemistry.EntitySystems;
+using Content.Shared.Anomaly.Components;
+using Robust.Shared.Random;
+using Content.Shared.Chemistry.Reagent;
+using Robust.Server.GameObjects;
+using Content.Shared.Chemistry.Components;
+using Content.Server.Fluids.EntitySystems;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Anomaly.Effects;
+/// <summary>
+/// This component allows the anomaly to chase a random instance of the selected type component within a radius.
+/// </summary>
+public sealed class LiquidAnomalySystem : EntitySystem
+{
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
```

Comment:
sort also rename _solutionContainerSystem to _solutionContainer and _puddleSystem to _puddle its nicer